service/status-report/main.go

```diff
@@ -215,6 +221,29 @@ func setClusterInformation(ctx context.Context, cl client.Client, status interfa
 
 }
 
+func setStorageQuotaUtilizationRatio(ctx context.Context, cl client.Client, status interfaces.StorageClientStatus) {
+	clusterResourceQuota := &quotav1.ClusterResourceQuota{}
+	clusterResourceQuota.Name = status.GetClientName()
+
+	// No need to check for NotFound because unlimited quota client will not have CRQ resource
+	if err := cl.Get(ctx, client.ObjectKeyFromObject(clusterResourceQuota), clusterResourceQuota); client.IgnoreNotFound(err) != nil {
+		klog.Warningf("Failed to get clusterResourceQuota %q: %v", clusterResourceQuota.Name, err)
+	}
+
+	if clusterResourceQuota.Status.Total.Hard != nil {
+		total := clusterResourceQuota.Status.Total.Hard["requests.storage"]
+		used := clusterResourceQuota.Status.Total.Used["requests.storage"]
+		ratio := used.AsApproximateFloat64() / total.AsApproximateFloat64()
+		if ratio > 1 {
+			status.SetStorageQuotaUtilizationRatio(1)
+		} else {
+			status.SetStorageQuotaUtilizationRatio(float32(ratio))
```

Comment:
are we going to show only ratio? and not capacity utilized

Reply:
@rohan47 Because this is not about ODF capacity utilization, the quota reports the number of bytes out of the quota for total cluster storage utilization. I made the call to use ratio because we are talking about a number that describes how close we are to the quota, I agree that the world utilization here is problematic, but we couldn't find better terminology. If you have a suggestion for a more fitting terminology, please reply